src/acparser/__init__.py

```diff
@@ -44,54 +47,85 @@ def system(cmd):
 
 def main():
     "Entry point"
-    namespace = sys.argv[1]
-    collection_name = sys.argv[2]
-    collection_version = sys.argv[3]
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--tarfile", help="Path to the source tarball", required=True)
+    parser.add_argument(
+        "--namespace", help="Namespace of the collection", required=True
+    )
+    parser.add_argument("--name", help="Name of the collection", required=True)
+    parser.add_argument("--version", help="Version of the collection", required=True)
+    args = parser.parse_args()
+    namespace = args.namespace
+    collection_name = args.name
+    collection_version = args.version
+
+
+    EXISTS_GALAXY = False
+    REQUIRES_ANSIBLE = ""
+    LICENSE = ""
+    CHANGELOG = ""
+    REQUIREMENTS = []
+    COMMUNITY_COLLECTIONS = ""
```

Comment:
These appear to all be used as local variables in specific functions now.  So I would:
* Remove the initialization of the values from here.
* Lowercase the variable names (UPPERCASE is used for constants and sometimes globals.  These are local variables.)
* When necessary, initialize the variables in the functions where they are used.